.gitignore

```diff
@@ -0,0 +1,2 @@
+env
+__pycache__
```

Comment:
pycache is still being tracked by your github please ensure to address this issue